### ISSUE TITLE
1.6.1 incompatible with poise

### ISSUE THREAD

**Issue by brianjmurrell** [chef-cookbooks/firewall#105]
When I try to use firewall 1.6.1 cookbook on Chef 11 I get:

```
[2015-10-28T09:37:44-07:00] ERROR: Halite is not compatible with no_lazy_load false, please set no_lazy_load true in your Chef configuration file.
```

But in https://github.com/poise/poise/issues/12#issuecomment-113586185 @coderanger says that poise is not compatible with Chef 11.

How does this all square?  Surely this combination worked at some point.

**Comment by coderanger** [comment]
You need to use Poise 1.x for Chef 11.

**Comment by martinb3** [comment]
Hi @brianjmurrell -- I talked to @coderanger and it sounds like poise 2.x dropped compatibility with chef 11.x at some point after the initial release. I'm doing a 1.6.2 maintenance release now which should address your issue. Cheers!

**Comment by martinb3** [comment]
Released in v1.6.2.
